sufsara_tutorial.Rmd

```diff
@@ -39,48 +65,28 @@ Press OK, and from here on you can press the bubble on the right-hand side to la
 
-<!--### "User" view
-
-Then on the SURFsara website, select your VM, press the garbage can button, and select *Shutdown*. -->
-
 ## Saving changes
 
 If you change something, then restart the VM, you may notice that your changes have been lost. This is because by default VM hard drives are not persistent. You need to clone them and set them as persistent to make the changes stick.
 
-<!--### "Cloud" view-->
-
 To do that, when your VM is in the *OFF* state, press the green floppy disk icon. Give your new template a descriptive name (for example, your name or the name of your group) and choose "Persistent". Wait a while while your changes are saved.
 
 From here on, as long as you start the VM from the template in the *Saved* tab, the changes you do will be saved.
 
 **Important**: Do not make more than one VM persistent, as it might go over the hard disk quota! If you want to start fresh, make sure your previous saved template is deleted first.
 
-<!--### "User" view
-
-To do that, go to the *Virtual Resources* → *Images* page on the SURFsara website. This is a list of hard disks available to your VMs. Select the *Ubuntu 16.04 GeoScripting* image, which is the operating system image, then click "Clone" and name it. Go back to the image listing, selected your cloned OS image, press on the three-dots button and choose "Make persistent".
-
-You now have a persistent clone of the operating system. Now you need to tell the VM to use it. To do that, go to the *Templates* page, select the template and clone it. Select your cloned template and click *Update*. Go to the *Storage* tab and click on your cloned OS image, then click *Update* to confirm changes.
-
-Now when you instantiate your cloned template, it will use the persistent image and allow you to make changes. In case something ever goes wrong with the VM, you can always delete the cloned image and reclone the original image to start fresh.
-
-The OS image only has 10 GiB of space. In case it is not enough, such as for processing large rasters, you can add an additional hard drive to your VM and store your files on it.
-
-To do that, repeat the cloning and template editing process, but this time with the *userdata shared drive* image. When editing your teplate file (you can keep updating the cloned template you already have), instead of replacing Disk 0, press the *Add another disk* button and choose your cloned *User data* image as Disk 1.
-
-The space is then available in the *userdata* directory within your user directory (*/home/ubuntu/userdata*).-->
-
 ## File sharing
 
 You might want to transfer files to or from the VM. There are two ways to do so. The more convenient method is to use *Network File System* (NFS) to access the VM drive as if it was a local hard drive. To do that, you need to establish a secure connection to the VM and map the network drive to your PC.
```

Comment:
It's probably a good idea to say something about file persistence to avoid losing them. There is no link to the M drive on SRC, and so the best way of transferring files (ones that are not already on GitLab, that is) is by using `sftp`, including a graphical client such as WinSCP.